### PR TITLE
[SIMPLE_FORMS] fix: 21-4138 - update suburls and page names

### PR DESCRIPTION
**Pull request by pennja**
## Summary

- Update suburls and page names for 21-4138
- Configured form 21-4138 to utilize new custom link logic
- Team: Veteran Facing Forms
- Flipper not implemented

## Related issue(s)

- department-of-veterans-affairs/va.gov-team#82269

## Testing done

- Local browser functions successfully
- Unit tests successful
- E2E tests successful

## What areas of the site does it impact?

This form ONLY.

### CODE REVIEW

File: src/applications/simple-forms/21-4138/pages/nameAndDateOfBirth.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
 import { getFullNameLabels } from '../helpers';
 
 /** @type {PageSchema} */
-export const nameAndDateOfBirthPage = {
+export const personalInformationPage = {
```
Comment (Thrillberg) [inline src/applications/simple-forms/21-4138/pages/nameAndDateOfBirth.js:12]: Do we want the page to mismatch the file name like this?

**Review by Thrillberg — DISMISSED** [review]
I definitely like switching out `pp` with more descriptive words but not sure all these renamings are necessarily improvements (not to say I'm against it, I'm just neutral)? But if you prefer the names this way, then 👍 !

This doesn't change any logic, right?

File: src/applications/simple-forms/21-4138/pages/nameAndDateOfBirth.js (hunk 1 above)
Comment (pennja) [reply src/applications/simple-forms/21-4138/pages/nameAndDateOfBirth.js:12]: Good catch, fixed! 👍

**Comment by pennja** [comment]
> I definitely like switching out `pp` with more descriptive words but not sure all these renamings are necessarily improvements (not to say I'm against it, I'm just neutral)? But if you prefer the names this way, then 👍 !
> 
> This doesn't change any logic, right?

@Thrillberg Yeah, this doesn't change any logic. I'm just trying to standardize the page names with the routes that CAIA wants. I figured it'd be better for anyone who has to touch this code in the future.

**Review by rhasselle-oddball — APPROVED** [review]

**Pull request merged**